softioc/device.py

```diff
@@ -219,14 +233,23 @@ def get(self):
         return self._value
 
 
-def _Device(Base, record_type, mlst=False, default=0, convert=True):
+def _Device(
+        Base,
+        record_type,
+        mlst=False,
+        default=0,
+        convert=True,
+        value_to_epics=None):
```

Comment:
Looking at how the instances of this class are coming out, I'd be tempted to make `convert` and `value_to_epics` positional arguments without default (and let's rename `convert` to `epics_rc` and pass either `EPICS_OK` or `NO_CONVERT`).

Reply:
`value_to_epics` is now a positional argument (as of b615f6b). Do you still want the changes here for `convert`?